python/src/aiconfig/default_parsers/openai.py

```diff
@@ -349,3 +349,45 @@ def refine_chat_completion_params(model_settings):
             completion_data[key] = model_settings[key]
 
     return completion_data
+
+
+def add_prompt_as_message(
+    prompt: Prompt, aiconfig: "AIConfigRuntime", messages: list, params=None
```

Comment:
`list` is a constructor; you want typing.List

Reply:
updated